### ISSUE TITLE
8LTS: MetaTags: URL view does not show entries

### ISSUE THREAD

**Issue by thomaszbz** [mblaschke/TYPO3-metaseo#395]
**MetaSEO version**: 3.0.0-dev
**TYPO3 version**: 8.7.0
**PHP version**: 7.0.16

URL view does not show entries when RealURL 2.2.0 is installed OR compatibility7 is NOT installed.

How to reproduce:

**Reproducing step 1:** 

- Make sure entries exist in the MetaTag section:

![auswahl_185](https://cloud.githubusercontent.com/assets/11200727/24848545/79836642-1dc7-11e7-87b4-78b93243df7c.png)

**Reproducing step 2:**

- Install compatibility7 extension.
- Do NOT install RealUrl.
- Keep DB-Schema up-to-date (compare via install tool)
- Flush caches
- Result: URL entries are shown (only some fields are shown when RealURL is not installed. That's intentional):

![auswahl_187](https://cloud.githubusercontent.com/assets/11200727/24848773/bf73a9d6-1dc8-11e7-9b77-15d03f1418fa.png)

Deprecation log does not report anything special here. Apache log stays clean.

**Reproducing step 3:**

- Uninstall compatibility7 extension (it's _very_ incompatible with RealUrl)
- Do NOT install RealUrl.
- Keep DB-Schema up-to-date (compare via install tool)
- Flush caches
- Result: URL view is empty now!

![auswahl_186](https://cloud.githubusercontent.com/assets/11200727/24848680/3199ebf2-1dc8-11e7-8b0b-62175a72b6f4.png)

**Reproducing step 4:**

- Uninstall compatibility7 extension
- Install RealUrl and Dependencies
- Repair your broken backend as needed (that's _not_ related to metaseo!)
- Keep DB-Schema up-to-date (compare via install tool)
- Flush caches
- Result: URL view is empty now!

![auswahl_186](https://cloud.githubusercontent.com/assets/11200727/24848680/3199ebf2-1dc8-11e7-8b0b-62175a72b6f4.png)

To sum it up: As soon as compatibility7 is _not_ installed, the Url view is empty. This is _not_ related to RealUrl in the first place.

That said: To fix this issue, uninstall compatibility7 AND uninstall RealUrl. That's the easiest way to reproduce this issue, without considering additional issues which might occur with RealUrl installed (metaseo uses a lot of tweaks for the URL simulation, which is disabled when RealUrl is not installed).

**Comment by thomaszbz** [comment]
Update:

- With  compatiblity7 _disabled_, it just _still_ worked.
- I even could install RealUrl 2.2.0, and it _still_ worked:

![auswahl_188](https://cloud.githubusercontent.com/assets/11200727/24851753/31441602-1dd5-11e7-9815-36cda6b5b404.png)

Even URL simulation works (thereby: don't click the icon, but next to it. The icon shows no activity via #394 .):

![gh1tnsgbtv](https://cloud.githubusercontent.com/assets/11200727/24851959/e4679f24-1dd5-11e7-982c-f43fb5784012.png)

**Comment by thomaszbz** [comment]
Duplicates #411: The url_schema attribute got removed.
